modules/ztp-lvms-installing-lvms-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-lvms-installing-using-web-console_{context}"]
+= Installing {lvms} by using the web console
+
+You can use the {product-title} web console to install {lvms-first}.
+
+.Prerequisites
+
+* Install the latest version of the {rh-rhacm} Operator.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+. Search for the *{lvms}* from the list of available Operators, and then click *Install*.
+. Keep the default selection of *Installation mode* [*"All namespaces on the cluster (default)"*] and *Installed Namespace* (*"openshift-operators"*) to ensure that the Operator is installed properly.
```

Comment:
Is there a reason why you use brackets for one parenthetical item and parentheses for the other? 🤔